packages/extension/src/components/organisms/TransactionListing/format.util.ts

```diff
@@ -1,4 +1,4 @@
-import { unix } from 'moment';
```

Comment:
Why did you remove moment.js?

Reply:
Because it's a deprecated library which is not supported by vite, dayjs is the new go-to